onnxscript/ir/serde.py

```diff
@@ -711,7 +712,6 @@ def deserialize_function(proto: onnx.FunctionProto) -> _core.Function:
         overload=getattr(proto, "overload", ""),
         graph=graph,
         attributes=typing.cast(List[_core.Attr], attributes),
```

Comment:
We should maintain the api

Reply:
Ref: https://github.com/microsoft/onnxscript/commit/dff5381a113783012daa63bef56a24119b7669ae

Reply:
Could you check if the modification meets your expectation?

Reply:
I think we should revert changes in this file, since it is the call site, which should rename unchanged from the changes we make in the implementation